mpd/tests.py

```diff
@@ -1172,6 +1170,12 @@ def readline(self):
         # directly passing around the awaitable
         return self._output.get()
 
+    async def readexactly(self, length):
```

Comment:
```suggestion
    async def read_exactly(self, length):
```

Reply:
This is just mocking something that's provided in the Python API (<https://docs.python.org/3/library/asyncio-stream.html#asyncio.StreamReader.readexactly>)